it/uk/gov/hmrc/transitmovements/connectors/ObjectStoreConnectorSpec.scala

```diff
@@ -0,0 +1,73 @@
+/*
+ * Copyright 2023 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import akka.NotUsed
+import akka.stream.scaladsl.Sink
+import akka.stream.scaladsl.Source
+import akka.util.ByteString
+import org.mockito.ArgumentMatchers.any
+import org.mockito.Mockito.when
+import org.scalatest.concurrent.ScalaFutures
+import org.scalatest.matchers.must.Matchers
+import org.scalatest.wordspec.AnyWordSpec
+import org.scalatestplus.mockito.MockitoSugar
+import uk.gov.hmrc.objectstore.client.Md5Hash
+import uk.gov.hmrc.objectstore.client.Object
+import uk.gov.hmrc.objectstore.client.ObjectMetadata
+import uk.gov.hmrc.objectstore.client.Path.File
+import uk.gov.hmrc.objectstore.client.play.PlayObjectStoreClient
+import uk.gov.hmrc.transitmovements.connectors.BaseUnitSpec
+import uk.gov.hmrc.transitmovements.connectors.ObjectStoreConnectorImpl
+
+import java.time.Instant
+import java.util.UUID.randomUUID
+import scala.concurrent.Future
+import scala.util.Try
+
+class ObjectStoreConnectorSpec extends AnyWordSpec with Matchers with MockitoSugar with ScalaFutures with BaseUnitSpec {
```

Comment:
I realise that this is a connector, and I realise I've told you to take it out, but these aren't integration tests anyway -- they are unit tests with mocked interfaces. These would really be unit tests.